src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -81,5 +81,9 @@ internal static class IDEDiagnosticIds
         public const string NamingRuleId = "IDE1006";
         public const string UnboundIdentifierId = "IDE1007";
         public const string UnboundConstructorId = "IDE1008";
+
+        public const string DeclaredVariableNeverUsedDiagnosticId = "CS0168";
+        public const string AssignedVariableNeverUsedDiagnosticId = "CS0219";
```

Comment:
This is not appropriate.  This file is the diagnostic IDs that the IDE is declaring.  IT's the ones our own analyzers will report.  Thse two diagnostics are compiler diagnostics.

Reply:
? what is this for?